### ISSUE TITLE
Remove fork status

### ISSUE THREAD

**Issue by torfsen** [wo-ist-markt/wo-ist-markt.github.io#32]
TL;DR: I suggest that we remove the fork dependency to the original repository at CodeForKarlsruhe.

I first started work on what is now wo-ist-markt.de at the OK Lab Karlsruhe, and the [initial repository](https://github.com/CodeforKarlsruhe/wo-ist-markt) was created in our GitHub organization. After @johnjohndoe's contributions and the decision to open the app up to many more cities this new repository was created as a fork of the original one. The original repository now only contains [a reference to this repository](https://github.com/CodeforKarlsruhe/wo-ist-markt/blob/gh-pages/README.md) and no further contributions are planned for it.

I therefore suggest that we cut the ties to the original repository. The fork status just adds misleading clutter to the GitHub UI (for example, if you create a pull request GitHub suggests a PR to the original repository). I would keep the original repository and its reference to the new location, the two repositories just wouldn't be fork-linked anymore.

Removing the fork status [can only be done manually by the GitHub support team](https://stackoverflow.com/a/16052845/857390) but it seems that it's usually not a problem.

What do you think?

**Comment by torfsen** [comment]
Any thoughts, @johnjohndoe?

**Comment by torfsen** [comment]
I've asked GitHub support to detach this repository.

**Comment by torfsen** [comment]
The repository has been detached.

**Comment by johnjohndoe** [comment]
Awesome.
